packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts

```diff
@@ -14,35 +14,33 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { injectable, postConstruct } from 'inversify';
-import { isOSX, isWindows } from '../../common/os';
+import { injectable, postConstruct, inject } from 'inversify';
+import { IKeyboardLayoutInfo } from 'native-keymap';
```

Comment:
it's just type right? it's erased from generated js? `native-keymap` dependency is not installed for the browser target

Reply:
Yes, it's just an interface. I reference that from several files. Should we move the type definition to `core/src/common` to avoid the warning?

Reply:
i'm fine with any way. Keeping in one place could be better that noone tries to import in other places. Maybe adding a comment would help.